oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/IndexController.java

```diff
@@ -161,6 +184,22 @@ public static List<ModelColumn> getPhysicalTableColumns(Model model) {
             return new ArrayList<>(PHYSICAL_INDICES_COLUMNS.get(tableName).values());
         }
 
+        public static String getPhysicalColumnName(String modelName, String columnName) {
```

Comment:
```suggestion
        @Deprecated
        public static String getPhysicalColumnName(String modelName, String columnName) {
```

Let's declare this method `@Deprecated` too. And put comments about this method should only be used when the context involves the old column names